test/fw/ptl/utils/plugins/ptl_test_loader.py

```diff
@@ -191,7 +194,30 @@ def prepareTestLoader(self, loader):
 
         def check_loadTestsFromNames(names, module=None):
             rv = old_loadTestsFromNames(names, module)
-            self.check_unknown()
+            test_list = self.__tests_list_copy
+            test_suites = self.__tests_list_copy['__only__ts__']
+            if (len(test_list) > 2) or (len(test_suites) != 0):
+                if 'PBSTestSuite' not in test_suites:
```

Comment:
why do you need this if condition? what if the user is trying to run `pbs_benchpress -i -t PBSTestSuite` when PTL_TESTS_DIR is set to a valid non-install directory

Reply:
When pbs_benchpress is run with -f option where file is not present in current directory then, self.__test_list_copy will have PBSTestSuite in self.__tests_list_copy['__only__ts__'] and it will go in if condition and try to run all of the test suites which is not expected. I have put this condition to skip for -f option.

Reply:
When I run this below command 
"pbs_benchpress -i -t PBSTestSuite" , it is giving output of all methods present in ptl/utils/pbs_testsuite.py